src/view/sdlview/sdlview.h

```diff
@@ -0,0 +1,19 @@
+#ifndef EVE_SDLVIEW_H
+#define EVE_SDLVIEW_H
+
+#include <SDL2/SDL.h>
+#include <SDL2/SDL2_gfxPrimitives.h>
+
+#include "../../model/world/world.h"
+
+class SDLView {
+private:
+    void setupsdl(const World & world);
+    void draw(const World & world);
+    void stop();
+    SDL_Window * window;
+    SDL_Renderer * renderer;
+public:
+    int render(const World & world);
```

Comment:
I suggest changing the return type to bool (see above)

Reply:
It was something I considered, I think the reason why I didn't was different error codes though I don't think I actually ended up going for those

Reply:
3f4a19d103d8272eba6bc22740cedf781d40af86